### PR TITLE
[PM-9891] [PM-8794] Update "auto-fill" to "autofill"

### PR DESCRIPTION
**Pull request by alec-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-8794

## 📔 Objective

Replace instances of "auto-fill" and "auto-filled" with "autofill" and "autofilled".

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by djsmith85 — DISMISSED** [review]

**Comment by alec-livefront** [comment]
@djsmith85 Seeing "User alec-livefront does not have the necessary access for this repository." - would it be possible for you to start the workflows for this PR?

**Comment by djsmith85** [comment]
> @djsmith85 Seeing "User alec-livefront does not have the necessary access for this repository." - would it be possible for you to start the workflows for this PR?

Runs have completed, just waiting on a review from @bitwarden/dept-architecture

File: docs/architecture/deep-dives/event-logs/index.md
Comment (withinfocus) [inline docs/architecture/deep-dives/event-logs/index.md:16]:
```suggestion
| Client | User interactions on locally synced data that does not result in server-side operations. These are typically prefixed with a `Client_` prefix. One example is `Cipher_ClientAutofill` which logs the autofill of organization items. |
```

**Review by withinfocus — DISMISSED** [review]

**Review by withinfocus — APPROVED** [review]

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**
